spec/pg/gc_compact_spec.rb

```diff
@@ -0,0 +1,85 @@
+# -*- rspec -*-
```

Comment:
This file seems to use tabs, but all other specs use the conventional 2-spaces indent, could you re-indent?

Reply:
No, all ruby-pg source files use tabs indent (for some reason).

Reply:
Oh, I mixed with FFI, sorry.